tools/licenses/lib/main.dart

```diff
@@ -1021,6 +1021,8 @@ class _RepositoryDirectory extends _RepositoryEntry implements LicenseSource {
     return !entry.fullName.endsWith('third_party/gn') &&
            !entry.fullName.endsWith('third_party/gradle') &&
            !entry.fullName.endsWith('third_party/imgui') &&
+           !entry.fullName.endsWith('third_party/tinygltf/examples') &&
```

Comment:
GLTF isn't going to be used at runtime in any vended tool right now. Let's ignore the whole thing.

Reply:
Done.